Kingfisher/ImageCache.swift

```diff
@@ -111,8 +114,13 @@ public class ImageCache {
         let cacheName = cacheReverseDNS + name
         memoryCache.name = cacheName
         
-        let paths = NSSearchPathForDirectoriesInDomains(.CachesDirectory, NSSearchPathDomainMask.UserDomainMask, true)
-        diskCachePath = (paths.first! as NSString).stringByAppendingPathComponent(cacheName)
+        if let dcp = path {
+            diskCachePath = dcp
+        }
+        else {
+            let paths = NSSearchPathForDirectoriesInDomains(.CachesDirectory, NSSearchPathDomainMask.UserDomainMask, true)
```

Comment:
Length Violation: Line should be 100 characters or less: currently 122 characters